### PR TITLE
Launch flexible_bitrate experiment to 50%.

### PR DESCRIPTION
**Pull request by gmajoulet**
Launch `flexible-bitrate` experiment to 90%.

### CODE REVIEW

**Comment by newmuis** [comment]
Should we go `10% -> 50% -> 100%` instead of `10% -> 90% -> 100%`?

File: build-system/global-configs/canary-config.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "doubleclickSraReportExcludedBlock": 0.1,
   "dfp-render-on-idle-cwv-exp": 1,
   "flexAdSlots": 0.05,
-  "flexible-bitrate": 0.1,
+  "flexible-bitrate": 0.9,
```
Comment (gmajoulet) [inline build-system/global-configs/canary-config.json:13]:
```suggestion
  "flexible-bitrate": 0.5,
```

File: build-system/global-configs/prod-config.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "doubleclickSraExp": 0.01,
   "doubleclickSraReportExcludedBlock": 0.1,
   "flexAdSlots": 0.05,
-  "flexible-bitrate": 0.1,
+  "flexible-bitrate": 0.9,
```
Comment (gmajoulet) [inline build-system/global-configs/prod-config.json:12]:
```suggestion
  "flexible-bitrate": 0.5,
```

**Comment by gmajoulet** [comment]
PTAL @newmuis

**Review by newmuis — APPROVED** [review]

**Pull request merged**